gato/gatos/MapleGato.py

```diff
@@ -0,0 +1,118 @@
+from ABaseGato import ABaseGato, require_alive
+
+
+class MapleGato(ABaseGato):
+    SELF_BUFF_CURRENCY_GAIN_REQUIRED: int = 10
+    SELF_ENERGY_LOSS_REDUCTION_PERCENTAGE_PER_STACK: int = 10
+    SELF_BUFF_MAX_STACKS: int = 4
+    SELF_ADDITIONAL_ENERGY_LOSS_REDUCTION_PERCENTAGE_PER_STACK_PER_EIDOLON: int = 2
+
+    TEAM_BUFF_CURRENCY_GAIN_REQUIRED: int = 40
+    TEAM_BUFF_MOOD_BOOST_PERCENTAGE: int = 10
+    TEAM_BUFF_MOOD_BOOST_CAP: int = 100
+    TEAM_BUFF_EFFICIENCY_BOOST_PERCENTAGE: int = 10
+    TEAM_BUFF_EFFICIENCY_BOOST_DURATION_MINS: int = 10
+    TEAM_BUFF_ADDITIONAL_EFFICIENCY_BOOST_DURATION_E6: int = 5
+
+    __doc__ = f"""
+        > Fav Food:
+        > Own energy consumption is decreased by {SELF_ENERGY_LOSS_REDUCTION_PERCENTAGE_PER_STACK}% everytime they gain {SELF_BUFF_CURRENCY_GAIN_REQUIRED} amount of plum blossoms (currency) (can stack up to {SELF_BUFF_MAX_STACKS} times). 
+        > Everytime {TEAM_BUFF_CURRENCY_GAIN_REQUIRED} amount of plum blossoms (currency) is gained, restore all allies mood by {TEAM_BUFF_MOOD_BOOST_PERCENTAGE}% and boost efficiency by {TEAM_BUFF_EFFICIENCY_BOOST_PERCENTAGE}% for {TEAM_BUFF_EFFICIENCY_BOOST_DURATION_MINS} minutes (a total amount of {TEAM_BUFF_MOOD_BOOST_CAP} mood can be restored).
+        > Eidolons: e1 -> e5: Own energy consumption decrease effect is enhanced by {SELF_ADDITIONAL_ENERGY_LOSS_REDUCTION_PERCENTAGE_PER_STACK_PER_EIDOLON}% per eidolon.
+        > e6: Efficiency boost lasts for an additional {TEAM_BUFF_ADDITIONAL_EFFICIENCY_BOOST_DURATION_E6} minutes.
+    """
```

Comment:
Not sure if this jank would work with embeds.

Since all the values are placeholders from the doc, thought I'd attempt to do it like this to not forget updating code when we tweak values.